### PR TITLE
comments + config + md5 + pep8

### PR DESCRIPTION
**Pull request by xBartu**

### CODE REVIEW

File: Cafeteria/Rating/__init__.py [hunk 1]
```diff
@@ -20,11 +20,18 @@ def rateMenu(self, mealId):
 
         # if MongoDatabase.getMeal(mealId = mealId) == None:
         #     return "Error: Meal Doesn't Exist."
-
+        comment = request.values.get("comment") or None
         remoteIp = md5(request.remote_addr).hexdigest()
-        MongoDatabase().insertCafeteriaRatingByMealId(
-                ObjectId(mealId), rating, remoteIp, datetime.now()
-        )
+        # refactor by creating object and inserting id like php
```
Comment (borakrc) [inline Cafeteria/Rating/__init__.py]: can't we use only one function for that purpose instead?

File: Cafeteria/Rating/__init__.py [hunk 2]
```diff
@@ -20,11 +20,18 @@ def rateMenu(self, mealId):
 
         # if MongoDatabase.getMeal(mealId = mealId) == None:
         #     return "Error: Meal Doesn't Exist."
-
+        comment = request.values.get("comment") or None
```
Comment (borakrc) [inline Cafeteria/Rating/__init__.py]: what does this line do?

File: MongoDatabase.py [hunk 3]
```diff
@@ -41,14 +41,20 @@ def setCafeteriaMenu(self, allMealsInFile):
             assert isinstance(eachMeal, MealContainer)
             self.db['cafeteriaMenu'].update({'date':eachMeal.date}, eachMeal.toJson(), upsert=True)
 
-
-
+    # TODO refactor the next 3 functions!
```
Comment (borakrc) [inline MongoDatabase.py]: please enter the todo's to the trello board, not in the code.

File: MongoDatabase.py [hunk 4]
```diff
@@ -41,14 +41,20 @@ def setCafeteriaMenu(self, allMealsInFile):
...
-
-
+    # TODO refactor the next 3 functions!
     def insertCafeteriaRating(self, day, month, year, dinnerName, rating, remoteIp, datetime):
         self.db['cafeteriaRating'].insert({'day':day, 'month':month, 'year':year, 'dinnerName':dinnerName, 'rating':rating, 'remoteIp':remoteIp, 'lastUpdateTime': datetime})
 
     def insertCafeteriaRatingByMealId(self, mealId, rating, remoteIp, datetime):
         self.db['cafeteriaRating'].insert({'mealId':mealId, 'rating':rating, 'remoteIp':remoteIp, 'lastUpdateTime': datetime})
 
+    def insertCafeteriaRatingCommentByMealId(self, mealId, rating,
```
Comment (borakrc) [inline MongoDatabase.py]: why not insert an empty string instead? so we can use only one function when inserting to 'cafeteriaRating' collection.

File: main.py [hunk 5]
```diff
@@ -182,6 +182,19 @@ def cacheBooklets():
 def rootPage():
     return SiteMap.siteMapString
 
+@app.route("/radio/app_links")
+def radyoApps():
```
Comment (borakrc) [inline main.py]: this is already implemented

File: MongoDatabase.py (hunk 4 above)
Comment (xBartu) [inline MongoDatabase.py]: That's a good idea, however, we are following SOLID. So, I don't want to violate it.  If we don't, It's good to go.

File: Cafeteria/Rating/__init__.py (hunk 1 above)
Comment (xBartu) [inline Cafeteria/Rating/__init__.py]: Answered trough the another question.

File: Cafeteria/Rating/__init__.py (hunk 2 above)
Comment (xBartu) [inline Cafeteria/Rating/__init__.py]: It guarantees variable comment gets None if the request.values.get('comment') returns [], "", 0 etc.

File: MongoDatabase.py (hunk 3 above)
Comment (xBartu) [inline MongoDatabase.py]: sure thing for the next time!

**Pull request merged**